deploy/examples/import-external-cluster.sh

```diff
@@ -20,7 +16,10 @@ MON_ENDPOINT_CONFIGMAP_NAME=rook-ceph-mon-endpoints
 ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE
 ROOK_EXTERNAL_MAX_MON_ID=2
 ROOK_EXTERNAL_MAPPING={}
+RBD_STORAGE_CLASS_NAME=ceph-rbd
+CEPHFS_STORAGE_CLASS_NAME=cephFS
 ROOK_EXTERNAL_MONITOR_SECRET=mon-secret
+NAMESPACE=rook-ceph-external
```

Comment:
are we overriding NAMESPACE here, check line 16 we already have `ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE`

Reply:
this need to operator namespace, changing